relay-filter/src/interface.rs

```diff
@@ -108,3 +110,41 @@ impl Filterable for Replay {
         self.user_agent()
     }
 }
+
+impl Filterable for Span {
+    fn csp(&self) -> Option<&Csp> {
+        None
+    }
+
+    fn exceptions(&self) -> Option<&Values<Exception>> {
+        None
+    }
+
+    fn ip_addr(&self) -> Option<&str> {
+        None
+    }
+
+    fn logentry(&self) -> Option<&LogEntry> {
+        None
+    }
+
+    fn release(&self) -> Option<&str> {
+        self.data.value().and_then(|d| d.release.as_str())
+    }
+
+    fn transaction(&self) -> Option<&str> {
+        if self.is_segment.value().map_or(false, |&s| s) {
+            return self.description.as_str();
+        }
+
+        None
+    }
+
+    fn url(&self) -> Option<Url> {
+        None
+    }
+
+    fn user_agent(&self) -> Option<&str> {
+        None
+    }
```

Comment:
Let's add a field `user_agent.original` to `SpanData` and get the value from there: https://opentelemetry.io/docs/specs/semconv/attributes-registry/user-agent/